apps/auth/utils.py

```
@@ -0,0 +1,40 @@
+def create_jwt_token(data: Dict) -> str:
+    """Creates a JWT token"""
+    to_encode = data.copy()
+    to_encode.update({
+        "iat": datetime.now(timezone.utc),
+        "iss": ISSUER
+    })
+
+    return jwt.encode(to_encode, AUTH_KEY, algorithm=ALGORITHM)
+
+
+def decode_jwt_token(token: str) -> Dict:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)